lte/gateway/deploy/roles/magma_deploy/vars/all.yaml

```diff
@@ -0,0 +1,23 @@
+################################################################################
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+magma_pkgrepo_proto: http
+magma_pkgrepo_host: facebookconnectivity.jfrog.io
```

Comment:
We need to change that before releasing 1.5

Reply:
right. once new repo is ready, we can change it.